doc/install/dependencies.rst

```diff
@@ -413,18 +413,45 @@ The documentation requires LaTeX and Graphviz.  These are not
 Python packages and must be installed separately.
 
 * `Graphviz <http://www.graphviz.org/download>`_
-* a minimal working LaTeX distribution, e.g. `TeX Live <https://www.tug.org/texlive/>`_ or
+* LaTeX distribution, e.g. `TeX Live <https://www.tug.org/texlive/>`_ or
```

Comment:
```suggestion
* a LaTeX distribution, e.g. `TeX Live <https://www.tug.org/texlive/>`_ or
```

This is a very minor nit. Otherwise this looks good to me.